README.md

```diff
@@ -50,7 +50,7 @@ Open the terminal with Ctrl+Alt+T and type `shell`. If this command returns `ERR
 Then download and run the installation script below:
 
 ```bash
-curl -Ls git.io/vddgY | bash
+exec bash --init-file <(curl -Ls git.io/vddgY)
```

Comment:
Does this mean we don't need to type `bash` from the `crosh` shell prompt?

Reply:
It's possible? I don't think the `crosh` shell has an `exec` binary, though.

Reply:
I always type `shell` at the crosh prompt...

Reply:
Yeah, seems simpler to go with that.

Reply:
What is the advantage of doing it this way?  IOW, why the change?